Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -40,7 +40,7 @@ public abstract class ViewRenderer<TView, TNativeView> : VisualElementRenderer<T
 		event EventHandler _controlChanging;
 		event EventHandler _controlChanged;
 
-
+		bool _isElementOrControlEmpty => Element == null || Control == null;
```

Comment:
This is not the right casing for a method/property. Pascal case, no underscore, please!
```suggestion
		bool IsElementOrControlEmpty => Element == null || Control == null;
```